public/src/components/epicTests/epicTestEditor.tsx

```diff
@@ -301,63 +285,44 @@ class EpicTestEditor extends React.Component<EpicTestEditorProps, EpicTestEditor
         <div>
           <EditableTextField
             text={test.tagIds.join(",")}
-            onSubmit={this.onListChange("tagIds")}
+            onSubmit={this.onListChange('tagIds')}
             label="Display on tags:"
             helperText="Separate each tag with a comma"
             editEnabled={this.isEditable()}
           />
 
           <EditableTextField
             text={test.sections.join(",")}
-            onSubmit={this.onListChange("sections")}
+            onSubmit={this.onListChange('sections')}
             label="Display on sections:"
             helperText="Separate each section with a comma"
             editEnabled={this.isEditable()}
           />
 
           <EditableTextField
             text={test.excludedTagIds.join(",")}
-            onSubmit={this.onListChange("excludedTagIds")}
+            onSubmit={this.onListChange('excludedTagIds')}
             label="Excluded tags:"
             helperText="Separate each tag with a comma"
             editEnabled={this.isEditable()}
           />
 
           <EditableTextField
             text={test.excludedSections.join(",")}
-            onSubmit={this.onListChange("excludedSections")}
+            onSubmit={this.onListChange('excludedSections')}
             label="Excluded sections:"
             helperText="Separate each section with a comma"
             editEnabled={this.isEditable()}
           />
 
           <Typography variant={'h4'} className={classes.h4}>Audience</Typography>
 
-          <FormControl
-            className={classes.formControl}>
-              <InputLabel
-                className={classes.selectLabel}
-                shrink
-                htmlFor="locations-select-multiple-checkbox">
-                  Locations:
-              </InputLabel>
-              <Select
-                className={classes.select}
-                multiple
-                value={test.locations}
-                onChange={this.onLocationsChange}
-                input={<Input id="locations-select-multiple-checkbox" />}
-                renderValue={selected => (selected as string[]).join(', ')}
-                disabled={!this.isEditable()}
-              >
-                {Object.values(Region).map(region => (
-                  <MenuItem key={region} value={region} >
-                    <Checkbox checked={test.locations.indexOf(region) > -1} />
-                    <ListItemText primary={region} />
-                  </MenuItem>
-                ))}
-              </Select>
-          </FormControl>
+          <TargetRegionsSelector
```

Comment:
I really like all these new custom components making the parent less cluttered! 👍